src/components/Grid/Entry/Entry.tsx

```diff
@@ -1,14 +1,26 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
There is currently an issue with TypeScript and Fragments that I cannot track down, but using `<>` is not working nicely: https://github.com/microsoft/TypeScript/pull/35392